scripts/publish-client

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DIR=$(pwd)
+IPFS_CACHE=$DIR/ipfs-cache
+
+set -e;
+cd repos/aragon;
+
+npm run publish:major -- --publish-dir $IPFS_CACHE/@aragon/aragon --skip-confirmation --no-propagate-content --no-build;
```

Comment:
You'll want to use the [`build:local` command](https://github.com/aragon/aragon/pull/874) instead